client/verta/verta/_internal_utils/_pip_requirements_utils.py

```diff
@@ -324,12 +318,21 @@ def inject_requirement(requirements, library, version):
         Python library, e.g. "verta".
     version : str
         Desired version number, e.g. "0.20.0".
+    error_on_conflict : bool
```

Comment:
The convention I've been seeing for default values in NumPy docstrings:
```suggestion
    error_on_conflict : bool, default True
```